pkg/apis/duck/v1alpha1/pubsub_lifecycle.go

```diff
@@ -20,23 +20,44 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-// MarkTopicNotReady sets the condition that the PubSub Topic is not ready and why.
-func (s *PubSubStatus) MarkTopicNotReady(cs *apis.ConditionSet, reason, messageFormat string, messageA ...interface{}) {
+// MarkTopicFailed sets the condition that the PubSub Topic is False and why.
+func (s *PubSubStatus) MarkTopicFailed(cs *apis.ConditionSet, reason, messageFormat string, messageA ...interface{}) {
 	cs.Manage(s).MarkFalse(TopicReady, reason, messageFormat, messageA...)
 }
 
+// MarkTopicUnknown sets the condition that the PubSub Topic is Unknown and why.
+func (s *PubSubStatus) MarkTopicUnknown(cs *apis.ConditionSet, reason, messageFormat string, messageA ...interface{}) {
+	cs.Manage(s).MarkUnknown(TopicReady, reason, messageFormat, messageA...)
+}
+
 // MarkTopicReady sets the condition that the PubSub Topic is ready.
 func (s *PubSubStatus) MarkTopicReady(cs *apis.ConditionSet) {
 	cs.Manage(s).MarkTrue(TopicReady)
 }
 
-// MarkPullSubscriptionNotReady sets the condition that the PubSub PullSUbscription is
-// not ready and why.
-func (s *PubSubStatus) MarkPullSubscriptionNotReady(cs *apis.ConditionSet, reason, messageFormat string, messageA ...interface{}) {
+// MarkTopicNotConfigured sets the condition that the PubSub Topic has not yet been reconciled.
+func (s *PubSubStatus) MarkTopicNotConfigured(cs *apis.ConditionSet) {
+	cs.Manage(s).MarkUnknown(TopicReady, "TopicNotConfigured", "Topic has not yet been reconciled")
```

Comment:
Topic has not been configured instead of reconciled? Or is it OK as is? 
If not, update the comment as well

Reply:
I follow the naming convention of serving https://github.com/knative/serving/blob/ec457adcb1466abfcbc1c0a069e60285e769d135/pkg/apis/serving/v1alpha1/route_lifecycle.go#L72. I will update the comments.

Reply:
OK, I like the consistency... leave the reason as is then, and see if you can update the comments.
Thanks @capri-xiyue